app/views/user_mailer/reservation_confirmation.text.erb

```diff
@@ -8,5 +8,7 @@ This is a confirmation of your reservation for the following equipment:
 
 You will be notified when the equipment you reserved is available for checkout.
 
+<p>You can view your  <%= link_to 'reservation request here', reservation_url(@complete_reservation, only_path: false) %>.</p>
```

Comment:
I think we use the word "request" to refer specifically to requests for reservations that fail validations; I would make the link text "reservation here" instead. Also, you have that extra space between "your" and the ERB block.